app/src/Components/AdminPage/LogOutBtn/logOutBtn.css

```diff
@@ -1,12 +1,12 @@
 .logOutBtn {
-    position: absolute;
-    top: 3%;
-    right: 5%;
+    float: right;
+    margin-right: 40px;
     background-color: #F27424;
     color: #ffffff;
```

Comment:
Capitals on hex color defs for consistency.

Reply:
Thanks for this, done for all hex codes :)